github/graphql/client.go

```diff
@@ -247,6 +290,25 @@ func (c *client) QueryGQL(ctx context.Context, qu string, vars Vars) (Map, error
 	}
 }
 
+// TODO: appID should be acquired from the context
+func (c *client) getPkey(ctx context.Context) (*rsa.PrivateKey, error) {
+	c.mu.Lock()
+	defer c.mu.Unlock()
+	key, ok := c.pkeys[c.appID]
```

Comment:
```suggestion
	c.mu.RLock()
	key, ok := c.pkeys[c.appID]
	c.mu.RUnlock()
	if ok {
	   return key
	}
	c.mu.Lock()
	defer c.mu.Unlock()
	key, ok = c.pkeys[c.appID]
	if ok {
	   return key
	}
```
This is much faster.

Reply:
Done.